community/artifactory-ha/values.yaml

```diff
@@ -370,14 +547,14 @@ nginx:
 
   readinessProbe:
     enabled: true
+    path: '/artifactory/webapp/#/login'
     initialDelaySeconds: 60
     failureThreshold: 10
     timeoutSeconds: 10
     periodSeconds: 10
     successThreshold: 1
   ## The SSL secret that will be used by the Nginx pod
   # tlsSecretName: chart-example-tls
-  tlsSecretName:
   env:
     ssl: true
     # artUrl: "http://artifactory:8081/artifactory"
```

Comment:
Commented values cannot be edited from the ICP web UI; please uncomment with empty value